.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: bug
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior
+1. install '...'
+2. setup '....'
+3. run '....'
+   (when run `leapp`, use always the `--debug` option to provide more data)
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**System information (please complete the following information):**
+ - OS and version: [e.g. Fedora 29]
```

Comment:
I'd just ask for the output of `uname -a` here

Reply:
Maybe we should also ask for the leapp-repository NVRA or the git commit id hash if from repo?

Reply:
discussed with guys. uname is good thing but it will not tell you what system you think you are using. I would rather require both info.

Reply:
ok, so the output of `uname -a` and `cat /etc/redhat-release`?